packages/salesforcedx-vscode-soql/package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "salesforcedx-vscode-soql",
+  "displayName": "SOQL",
+  "description": "Provides code-editing features for SOQL",
+  "qna": "https://github.com/forcedotcom/salesforcedx-vscode/issues",
+  "bugs": {
+    "url": "https://github.com/forcedotcom/salesforcedx-vscode/issues"
+  },
+  "repository": {
+    "url": "https://github.com/forcedotcom/salesforcedx-vscode"
+  },
+  "version": "0.0.1",
+  "publisher": "salesforce",
+  "license": "BSD-3-Clause",
+  "engines": {
+    "vscode": "^1.46.0"
+  },
+  "categories": [
+    "Programming Languages"
+  ],
+  "dependencies": {
+    "@salesforce/soql-builder-ui": "0.0.2"
+  },
+  "devDependencies": {
+    "@types/node": "12.0.12",
+    "@types/vscode": "^1.46.0",
+    "typescript": "3.7.5"
+  },
+  "scripts": {
```

Comment:
I'd be good to add `"vscode:package": "vsce package"` so CircleCI starts building the vsix and you can start using it for testing.

Reply:
Good idea, thanks!